.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "cSpell.words": [
```

Comment:
I wonder if these settings should be in the repo or developer's own checkout.

Reply:
Ah, I missed that I had added this file :smile:. They are ones that are somewhat specific to this project (which is why I added them to the project workspace and not the global one). But having the `.vscode` folder in `.gitignore` also seems reasonable to me.